modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/DataRegionMetricsImpl.java

```diff
@@ -561,6 +568,12 @@ public void clear() {
         pageReplaceAge.reset();
     }
 
+    /** @param time Time to add to {@code totalThrottleTime} metric. */
```

Comment:
I think we should add to javadoc a measurement metric unit.

Reply:
Done.